README.md

```diff
@@ -1,68 +1,67 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+### unStack Africa Virtual Conference.  
 
-## Available Scripts
+![unStack Africa](./src/assets/img/logo.png)
```

Comment:
Change logo to "https://github.com/unStackOfficial/unStack-logo-assets/blob/master/Group%205.png"

Reply:
My apologies for the late response. School has just resumed and I am trying to get into school moods again.

Reply:
Changes made?